### DISCUSSION TITLE
Update ControlzEx from version 3.x to 4.3 and MahApps.Metro from version 1.65 to version 2.0

### DISCUSSION THREAD

**Discussion by Lajos4444** [ControlzEx/ControlzEx#196] [Q&A]
Hy, I am not a very experienced developer (recent graduate). My task is to update all dependencies of an application .
One dependency is ControlzEx. I updated ControlzEx from version 3.x to 4.3.
With this update, MahApps.Metro has also been updated from version 1.65 to version 2.0

The application was successfully built, and all functuon worked perfectly before i updated ControlzEx to version 4.3. After the update i got a lot of error, for example:

In App.xaml:

xmlns:vm="clr-namespace:Vulnerator.ViewModel"

Error in this file:
The name "ViewModelLocator" does not exist in the namespace "clr-namespace:Vulnerator.ViewModel".

ViewModelLocator.cs:

using CommunityToolkit.Mvvm.DependencyInjection;
using Microsoft.Extensions.DependencyInjection;

namespace Vulnerator.ViewModel
{
public class ViewModelLocator
{
public ViewModelLocator()
{
Ioc.Default.ConfigureServices(
new ServiceCollection()
.AddSingleton() ....

The namespace is correct, the file location is correct. I can not figure out how to solve this problems.
I tried to add the assembly name to the references.

xmlns:vm="clr-namespace:Vulnerator.ViewModel;assembly=Vulnerator" but it doesn't solved the problems.

I got the same errors for a lot of namespaces.
The name "DependencyProperties" does not exist in the namespace "clr-namespace:Vulnerator.View.ViewHelper".
The name "IsNullToCollapsedConverter" does not exist in the namespace "clr-namespace:Vulnerator.View.Converter".
etc.

The namespace, the location of all files are correct.

Environment
ControlzEx version: v4.3
Windows build number: Win10 Version 22H2 OS build 19045.4529
Visual Studio: Microsoft Visual Studio Community 2019 (2) Version 16.11.33
Target Framework: .Net Framework 4.7
Screenshots

![image](https://github.com/ControlzEx/ControlzEx/assets/135051257/24fccb35-0e2b-4d4f-84b6-4e4c09083d57)
![image](https://github.com/ControlzEx/ControlzEx/assets/135051257/15e66750-5c13-4c83-8b94-e2f07f174d33)
![image](https://github.com/ControlzEx/ControlzEx/assets/135051257/05aff18a-df86-45d2-9c66-e2accf7bf37c)
![image](https://github.com/ControlzEx/ControlzEx/assets/135051257/3f254a22-23ea-4478-b8a3-6475589c47fd)
![image](https://github.com/ControlzEx/ControlzEx/assets/135051257/12921080-d973-47f9-9ab6-27f5662b6128)

Thanks in advance for your help!

**Comment by batzen** [comment]
Could you try to build from the commandline and post the build log here?
